core/src/main/scala/com/databricks/labs/remorph/Main.scala

```diff
@@ -9,6 +9,8 @@ object Main extends App with ApplicationContext {
       exampleDebugger.debugExample(args("name"), args.get("dialect"))
     case Payload("debug-me", _) =>
       prettyPrinter(workspaceClient.currentUser().me())
+    case Payload("coverage", args) =>
```

Comment:
rename command to `debug-coverage` - that way customers won't be tempted to call it.